tools/mason/MasonPublish.chpl

```diff
@@ -78,7 +82,14 @@ proc masonPublish(ref args: list(string)) throws {
     }
     var createReg = createFlag.valueAsBool();
 
-    const badSyntaxMessage = 'Arguments does not follow "mason publish [options] <registry>" syntax';
+    const badSyntaxMessage = 'Arguments do not follow "mason publish [options] <registry>" syntax';
+
+    if refreshLicenses {
+      writeln("Force updating list of valid license names from SPDX repo...");
+      refreshLicenseList(true);
+      writeln("done updating license list");
```

Comment:
After grepping around, it looks like we tend to use capitalization for the first word.